### PR TITLE
docs: move clustering doc

### PR DESCRIPTION
**Pull request by xiang90**

### CODE REVIEW

**Comment by gyuho** [comment]
LGTM. Thanks.

File: Documentation/op-guide/clustering.md [hunk 1]
```diff
@@ -383,52 +364,17 @@ $ etcd --name infra2 \
...
-
-You might see an error like `cannot find local etcd $name from SRV records.`. That means the etcd member fails to find itself from the cluster defined in SRV records. The resolved address in `--initial-advertise-peer-urls` *must match* one of the resolved addresses in the SRV targets.
-
-# 0.4 to 2.0+ Migration Guide
-
-In etcd 2.0 we introduced the ability to listen on more than one address and to advertise multiple addresses. This makes using etcd easier when you have complex networking, such as private and public networks on various cloud providers.
+### Proxy
 
-To make understanding this feature easier, we changed the naming of some flags, but we support the old flags to make the migration from the old to new version easier.
+When `--proxy` flag is set, etcd will run in [proxy mode][proxy]. This proxy mode only supports v2 API. We have no plan to make the it support v3 API. Instead we will provide a new proxy after etcd 3.0 release to provide enhanced features and support v3 API.
```
Comment (heyitsanthony) [inline Documentation/op-guide/clustering.md]: When the `--proxy` flag is set, etcd runs in [proxy mode][proxy]. This proxy mode only supports the etcd v2 API; there are no plans to support the v3 API. Instead, for v3 API support, there will be a new proxy with enhanced features following the etcd 3.0 release.

**Comment by xiang90** [comment]
@heyitsanthony Fixed. PTAL.

**Comment by joshix** [comment]
LGTM

**Pull request merged**